pkgs/development/interpreters/php/generic.nix

```diff
@@ -33,7 +33,7 @@ let
     , jq
 
     , version
-    , hash
+    , hash ? ""
```

Comment:
Why not just pass empty hash up there? This will make it easier to accidentally omit.

Reply:
This is a totally opinionated change.

Reply:
This is an opinionated change, I just found it odd to have an empty hash in the file 8.3.nix.

Anyway, if the `hash` value is empty, will nix will complain when building, so, this is the reason.

Reply:
You could just set it to `null`. And having the `hash` argument optional will fail earlier, saving developer’s time, and produce a more informative trace.

Reply:
The default value is still here.